src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -478,12 +479,15 @@ uct_cuda_copy_md_mem_query(uct_md_h tl_md, const void *address, size_t length,
     }
 
     if (mem_attr->field_mask & UCT_MD_MEM_ATTR_FIELD_DMABUF_FD) {
-        mem_attr->dmabuf_fd = uct_cuda_copy_md_get_dmabuf_fd(&addr_mem_info);
-    }
-
-    if (mem_attr->field_mask & UCT_MD_MEM_ATTR_FIELD_DMABUF_OFFSET) {
-        mem_attr->dmabuf_offset = UCS_PTR_BYTE_DIFF(addr_mem_info.base_address,
-                                                    address);
+        dmabuf_addr_mem_info = addr_mem_info;
+        mem_attr->dmabuf_fd  = uct_cuda_copy_md_get_dmabuf_fd(&dmabuf_addr_mem_info);
```

Comment:
seems this change does not change the behavior because dmabuf_addr_mem_info is not modified by uct_cuda_copy_md_get_dmabuf_fd